profit-sharing.md

```diff
@@ -1,47 +1,58 @@
-# Profit Sharing
+# Profit sharing
 
-We believe every Nitean should take part in the distribution of profit. This document describes how we manage profit distribution.
+This document describes how we manage the distribution of the company profits.
 
+  **Contents**
+  1. [The concept](#the-concept)
+  2. [Past](#past)
+  3. [Present](#present)
+  4. [Future](#future)
+  5. [Donations](#donations)
 
-## The idea
 
-We divide the profit into three parts, based on how we perceive time: the past, the present and the future.
+## The concept
 
-Profit is distributed at the end of each quarter, based on the performance of the company.
+Any profit that the company has made is divided into three parts based on how we perceive time: the past, the present and the future. These temporal parts are 90% of the profit share and are split three-ways equally. This leaves a remaining 10% for donating to causes we believe in.
```

Comment:
Instead of `These temporal parts are 90% of the profit share and are split three-ways equally.` -> `All parts are split three-ways equally. To keep calculations simple we use 30% instead of 1/3 when calculating the parts. This leaves the remaining 10% for donating to causes we believe in.`

Reply:
The original reason why we even have donations is that calculations with 33% were vastly easier than thirds. I'd like to keep this reason.

Reply:
I understand that but now the reasoning has changed and it puts a more positive light on the donations as it's no longer 'to make accounting easier' but something we choose to do

Reply:
or.. 

`Profit is distributed at the end of each quarter, based on the performance of the company. We believe in donating to good causes and reserve 10% of the profit to hopefully make a big impact upon them. The remaining profit is split three-ways (30% per part) based on how we perceive time: the past, the present and the future. `

Reply:
👍